test/others/libcriu/run.sh

```diff
@@ -62,6 +62,16 @@ if [ "$(uname -m)" = "x86_64" ]; then
 fi
 run_test test_errno
 run_test test_join_ns
+if criu check --feature mem_dirty_track > /dev/null; then
+	export CRIU_FEATURE_MEM_TRACK=1
```

Comment:
Not sure I understand you correctly. If the feature is not enable, for example dirty page tracking will always be disabled on aarch64 because of missing kernel support, the environment is not set and then in `test_feature_check.c` it will not be set to true and so we will not check it. But that does mean it will not be checked at all. Hmm...

Ah, I think I understand. If it is disabled we should still check the feature and get the result that it is disabled. So I always have to check all features and compare the result with the result from `criu check`. Makes sense.